app/views/idv/review/new.html.slim

```diff
@@ -2,13 +2,19 @@
 
 h1.h3 = t('idv.titles.session.review')
 
-= render 'shared/user_verify_password', update_path: idv_review_path
```

Comment:
Looks like this partial is only used in one place now. That makes me think we may want to decompose it and use its code directly in the verify password template.